cmd/operator-controller/main.go

```diff
@@ -235,9 +235,9 @@ func main() {
 		LeaderElectionID:       "9c4404e7.operatorframework.io",
 		// Recommended Leader Election values
 		// https://github.com/openshift/enhancements/blob/61581dcd985130357d6e4b0e72b87ee35394bf6e/CONVENTIONS.md#handling-kube-apiserver-disruption
-		LeaseDuration: ptr.To(137 * time.Second),
-		RenewDeadline: ptr.To(107 * time.Second),
-		RetryPeriod:   ptr.To(26 * time.Second),
+		LeaseDuration: ptr.To(137 * time.Second), // Default: 15s
```

Comment:
are you just commenting them so we see how long they are compared to the defaults?

Reply:
It only makes it easier for us to know the default.
We do not need to check the source code. 
We can remove it. Just a comment to try make easier

Reply:
Removed 👍